models/Par0004/Par0004.bs_base.NRU08.yml

```diff
@@ -0,0 +1,70 @@
+#ImageTypes
+FixedImagePixelType: "short"
+FixedImageDimension: 3
+MovingImagePixelType: "short"
+MovingImageDimension: 3
+
+#Components
+Registration: "MultiResolutionRegistration"
+FixedImagePyramid: "FixedRecursiveImagePyramid"
+MovingImagePyramid: "MovingRecursiveImagePyramid"
+Interpolator: "BSplineInterpolator"
+Metric: "MattesMutualInformationWithRigidityPenalty"
+Optimizer: "StandardGradientDescent"
+ResampleInterpolator: "FinalBSplineInterpolator"
+Resampler: "DefaultResampler"
+Transform: "BSplineTransform"
+
+# Transform
+FinalGridSpacing: 8.0
+UpsampleGridOption: true
+
+# Multi resolution
+NumberOfResolutions: 4
+
+# Sampling
+NewSamplesEveryIteration: [true, true, true, true]
+NumberOfSpatialSamples: [5000, 5000, 5000, 5000]
+NewSamplesEveryIteration: [true, true]
```

Comment:
- With respect to issue #13, note that YAML _does_ allow parameters to be specified more than once. (As you see here, specifying `NewSamplesEveryIteration` twice.) Of course, it should still be detected by elastix, which would then produce an error message.